lib/rubocop/cop/style/keyword_arguments_order.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop enforces that optional keyword arguments are placed at the
+      # end of the arguments list.
+      #
+      # This improves readability, because when looking through the source,
+      # it is expected to find required arguments at the beginning of parameters list
+      # and optional arguments at the end.
+      #
+      # @example
+      #   # bad
+      #   def some_method(first: false, second:, third: 10)
+      #     # body omitted
+      #   end
+      #
+      #   # good
+      #   def some_method(second:, first: false, third: 10)
+      #     # body omitted
+      #   end
+      #
+      class KeywordArgumentsOrder < Base
```

Comment:
In this context it's parameters, not arguments - params in the definition, args in the call sites. The messages have to updated accordingly as well.